src/pvecontrol/config.py

```diff
@@ -12,16 +12,24 @@
             "password": confuse.Optional(str, None),
             "token_name": confuse.Optional(str, None),
             "token_value": confuse.Optional(str, None),
+            "timeout": confuse.Optional(int, default=0),
```

Comment:
With my previous config there an error when I let the default value, should we set it to something like 10 ?
```
ValueError: Attempted to set connect timeout to 0, but the timeout cannot be set to a value less than or equal to 0.
```

```suggestion
            "timeout": confuse.Optional(int, default=10),
```

Reply:
The default value is already configured in `config_default.yaml` and set to `60`.